stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint.py

```diff
@@ -61,6 +61,10 @@ def default_get(self, fields):
     def make_procurement(self):
         self.ensure_one()
         errors = []
+        # User requesting the procurement is passed by context to be able to
+        # update final MO, PO or trasfer with that information.
+        pg_obj = self.env['procurement.group'].with_context(
```

Comment:
@lreficent Is that cannot be done without requiring context change ? This can decrease performances.

Reply:
It would require several hooks into odoo code.

Reply:
Well, it is something manually triggered so it will only create a new environment with that context each time a user uses the wizard. I think it is not something critical in this case.